webpack.build.js

```diff
@@ -49,33 +46,54 @@ module.exports = {
     ]),
   ],
   resolve: {
-    extensions: ['', '.js', '.jsx', '.cjsx', '.coffee', '.styl', '.css'],
-    modulesDirectories: ['.', 'node_modules'],
+    extensions: ['.js', '.jsx', '.cjsx', '.coffee', '.styl', '.css'],
+    modules: ['.', 'node_modules'],
   },
   module: {
-    loaders: [{
+    rules: [{
       test: /\.jsx?$/,
       exclude: /node_modules/,
-      loader: 'babel',
+      use: 'babel-loader',
     }, {
       test: /\.cjsx$/,
       exclude: /node_modules/,
-      loaders: ['babel', 'coffee', 'cjsx'],
+      use: [{
+        loader:'babel-loader'
+      }, {
+        loader: 'coffee-loader'
+      }, {
+        loader: 'cjsx-loader'
+      }],
     }, {
       test: /\.coffee$/,
-      loaders: ['babel', 'coffee'],
-    }, {
-      test: /\.json?$/,
-      loader: 'json',
+      use: [{
+        loader: 'babel-loader'
+      }, {
+        loader: 'coffee-loader'
+      }],
     }, {
       test: /\.css$/,
-      loader: ExtractTextPlugin.extract('style-loader', 'css-loader?root=../public'),
+      use: ExtractTextPlugin.extract({
+        fallback: 'style-loader', 
+        use: {
+          loader: 'css-loader',
+          options: { root: '../public' }
+        }
+      }),
     }, {
       test: /\.styl$/,
-      loader: ExtractTextPlugin.extract('style-loader', 'css-loader?root=../public!stylus-loader'),
+      loader: ExtractTextPlugin.extract({
+        fallback: 'style-loader', 
+        use: [{
+          loader: 'css-loader'
+          options: { root: '../public' },
```

Comment:
This line is throwing an error when I try a staging build. Should this block match the same block in `webpack.dev.js`?